portal-web/docroot/html/taglib/ui/search_iterator/bottom.jspf

```diff
@@ -109,7 +109,10 @@
 
 						var fixedHeaderEventHandler = function(event) {
 							var scrollTop = verticalScrollingContainer.scrollTop;
-							var trDomRecTop = trDomRect.top;
+
+							var trDomRect = fixedHeader.previousElementSibling.getBoundingClientRect();
+
+							var trDomRecTop = (trDomRect.top + 70);
```

Comment:
70 kind of seems like a "magic number" here. Is there some specific reason to use 70 vs any other?

Reply:
Its through testing the scroll and finding which will make the header appear and disappear correcly. We can also do 80 which works as well.

Reply:
Right, but does that number 70/80 come from something specific? Like is it the height of the header or something?

Reply:
Not really. It is what works best when comparing the `trDomRecTop` and the scroll pixels.

When the scroll pixels reach 93, this is the "ideal" time to reveal the fixedHeader. At this point the `trDomRecTop` is 12pixels. So adding 80 will make it 92 pixel revealing the fixedHeader.

Let me know if I should try something else.

Reply:
Okay that should be fine. I was just making sure there wasn't some way we could infer that number instead of it being sort of a "best guess"